Chapter 11B/A Date Class.txt

```diff
@@ -0,0 +1,65 @@
+%Attempt at making something by Grace lol maybe we can make this a nice simple cody for OOP?
+
+%Create a class 'Date' that has three properties: Month (a string), Day and
+%Year (positive integers). Overload the disp to print the date in the
+%format as follows:
+ 
+%today=Date('April',20,2017)
+%The date is April 20, 2017.
+ 
+
+
+%Create your Date class below!
+
+%Reference Solution
+
+classdef Date
+    properties
+        Month

```

Comment:
Property names are typically uppercase on the first letter to differentiate between props and vars.